### DISCUSSION TITLE
Support for Tailwind v3

### DISCUSSION THREAD

**Discussion by discoverlance-com** [forsartis/vue-cli-plugin-tailwind#44] [General]
Is there any working progress or intent on support for tailwind recently released v3? 

Is it something to look out for this year or probably next year?

**Comment by forsartis** [comment]
You can try the beta. See [v3.0.0-beta.0](https://github.com/forsartis/vue-cli-plugin-tailwind/releases/tag/v3.0.0-beta.0) for instructions.
